src/Parser.php

```diff
@@ -190,7 +190,7 @@ public function parse(string &$data, string $encoding, string $url = '')
                         } else {
                             $attrName = $xml->localName;
                         }
-                        $attributes[$attrName] = $xml->value;
+                        $attributes[(string) $attrName] = (string) $xml->value;
```

Comment:
What error causes this cast? Neither `localName` or `value` are supposed to be anything but string https://www.php.net/manual/en/class.xmlreader.php

Reply:
@jtojnar, below is the error that casting these to string resolves.  Looks like this stems from the possible types of the properties accessed from `XMLReader`.

>   Line   src/Parser.php                                                                                                                  
> 195    Parameter #3 $attributes of method SimplePie\Parser::tag_open() expects array<string, string>, array<int|string, mixed> given.

Reply:
I've just pushed https://github.com/simplepie/simplepie/pull/857/commits/bb89c217acf9e2f3091e9820297f446f1379408f which moves this fix more upstream by only casting `localName` to string.  This also seems to fix this error 🤷‍♂️